components/sensor/airthings_ble.rst

```diff
@@ -0,0 +1,73 @@
+AirThings BLE Sensors
+========================
+
+.. seo::
+    :description: Instructions for setting up AirThings bluetooth-based sensors in ESPHome.
+    :keywords: AirThings, BLE, Bluetooth, Wave Plus
+
+The ``airthings_wave_plus`` sensor platform lets you track the output of AirThings Bluetooth Low Energy devices.
+
+This component will track radon, airborne chemicals, CO2, temperature and humidity and others depending on the device sensors.
+
+Device Discovery
+-----------------
+
+AirThings devices can be found using the ``airthings_ble`` ble scanner.
+
+To find out your device's MAC address, add the following to your ESPHome configuration:
+
+.. code-block:: yaml
+
+    logger:
+      level: DEBUG # Required for the tracker to show the device
+
+    esp32_ble_tracker:
+    airthings_ble:
+
+The device will then listen for nearby devices, and display a message like this one:
+
+``
+[D][airthings_ble:019]: 
+Found AirThings device Serial: 123456789 (MAC: 01:02:03:04:05:06)
+``
+
+Once the device is found, **remove the ``airthings_ble:`` device tracker** from your configuration and take note of the device MAC address, and use it when configuring a sensor below.
+
+Supported Devices
+-----------------
+
+Wave Plus
+*********
+
+AirThings Wave Plus tracks radon (24h and long term), airborne chemicals, CO2, temperature, atmospheric pressure and humidity.
+
+.. figure:: images/airthings_wave_plus.jpg
+    :align: center
+    :width: 60.0%
+
+Configuration example:
+
+.. code-block:: yaml
+
+    sensor:
+      - platform: airthings_wave_plus
+        mac_address: '01:02:03:04:05:06'
+        update_interval: 5min # default
```

Comment:
The `update_interval` option doesn't seem to be supported using develop/jela/waveplus@38dd289ea2fa5826d6c7c84ee98b1fcab5556a6e :

```
INFO Reading configuration airthings.yaml...
Failed config

sensor.airthings_wave_plus: [source detect.yaml:13]
  platform: airthings_wave_plus
  mac_address: XX:XX:XX:XX:XX:XX

  [update_interval] is an invalid option for [sensor.airthings_wave_plus]. Please check the indentation.
  update_interval: 5min [source detect.yaml:15]
  temperature:
    name: WavePlus Temperature
  radon:
    name: WavePlus Radon
  radon_long_term:
    name: WavePlus Radon Long Term
...
```

Reply:
Thanks for the review! This is fixed in https://github.com/esphome/esphome/pull/1656/commits/045a72d4c5350f6aa96ae3eb870ef1a23e6a7c8f